src/corelibs/U2Core/src/tasks/ExportTasks.cpp

```diff
@@ -223,31 +189,25 @@ void ExportMSA2MSATask::run() {
 
 ExportDNAChromatogramTask::ExportDNAChromatogramTask(DNAChromatogramObject *_obj, const ExportChromatogramTaskSettings &_settings)
     : DocumentProviderTask(tr("Export chromatogram to SCF"), TaskFlags_NR_FOSCOE),
-      cObj(_obj), settings(_settings), loadTask(nullptr) {
+      chromaObject(_obj), settings(_settings), loadTask(nullptr) {
     GCOUNTER(cvar, "ExportDNAChromatogramTask");
     setVerboseLogMode(true);
 }
 
 void ExportDNAChromatogramTask::prepare() {
-    Document *d = cObj->getDocument();
-    assert(d != nullptr);
-    if (d == nullptr) {
-        stateInfo.setError("Chromatogram object document is not found!");
-        return;
-    }
+    Document *d = chromaObject->getDocument();
+    SAFE_POINT_EXT(d != nullptr, stateInfo.setError("Chromatogram object document is not found!"), );
```

Comment:
`L10N::nullPointerError` could be used for this type of errors

Reply:
Done